R/get-grades.R

```diff
@@ -90,6 +94,19 @@ calculate_grades <- function(gs, policy){
   return (grades)
 }
 
+
+handle_excused <- function(grades_mat, assignments){
```

Comment:
Add tests for when there is an excused assignment and drops within a category.

Reply:
Done.